src/litserve/callbacks/defaults/metric_callback.py

```diff
@@ -0,0 +1,21 @@
+import time
+import typing
+from logging import getLogger
+
+from ..base import Callback
+
+if typing.TYPE_CHECKING:
+    from litserve import LitAPI
+
+logger = getLogger(__name__)
+
+
+class MetricLogger(Callback):
```

Comment:
Nit: the name is too broad. A metric might mean many things.

Reply:
updated to `PredictionTimeLogger`